components/todo-input.js

```diff
@@ -1,20 +1,96 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput(props) {
+	const { date, todo, onChangeTodo, onChangeDate, onSubmit } = props;
+
+	const maxLengthCheck = (object) => {
+		if (object.value.length > object.maxLength) {
+			object.value = object.value.slice(0, object.maxLength);
+		}
+	};
+
+	return (
+		<Form>
+			<InputSection>
+				<P>시간</P>
+				<Input
+					maxlength="8"
+					oninput="maxLengthCheck(this)"
+					type="number"
+					placeholder="날짜를 입력하세요 (ex.20200404)"
+					value={date}
+					onChange={onChangeDate}
+				/>
+			</InputSection>
+
+			<InputSection>
+				<P>TODO</P>
+				<TextArea
+					placeholder="할 일을 입력하세요"
+					value={todo}
+					onChange={onChangeTodo}
+				/>
+			</InputSection>
+			<Button onClick={onSubmit}>등록</Button>
+		</Form>
+	);
```

Comment:
```suggestion
	return (
		<Wrapper>
			<Row>
				<Label>시간</Label>
				<Input
					type="number"
					name="date"
					placeholder="날짜를 입력하세요 (ex.20200404)"
					onChange={handleFormChange}
				/>
			</Row>
			<Row>
				<Label>TODO</Label>
				<TextArea
				        name="todo"
					placeholder="할 일을 입력하세요"
					onChange={handleFormChange}
				/>
			</Row>
			<Button onClick={onSubmit}>등록</Button>
		</Wrapper>
	);
```
1. self-closing 태그 잘 사용해주셨네요 👍 
2. 컴포넌트 이름 바뀐 건 아래 리뷰들을 참고해주세요.
3. 위에 handleFormChange가 잘 작동하게 하기 위해 ```name``` attribute를 추가해줬어용
4. onInput 문법이 존재하긴 하지만 리액트 팀에서는 이것의 사용을 추천하지 않고 있습니다. [참고](https://thebook.io/006961/part01/ch07/01/01-01/)
onChange가 있는데 onInput을 또 써줄 필요는 없으므로 삭제해주시면 됩니다.
5. ```input```에 ```value```를 넣어줄 필요는 없어요! ```value```를 써주지 않았는데 ```handler```가 어떻게 ```e.target.value```를 파악하는지 궁금하다면 다음을 참고해주세요. [참고](https://thebook.io/006961/part01/ch07/01/01-01/)